libbeat/processors/fingerprint/hash.go

```diff
@@ -24,16 +24,23 @@ import (
 	"crypto/sha512"
 	"hash"
 	"strings"
+
+	"github.com/OneOfOne/xxhash"
 )
 
 type hashMethod func() hash.Hash
 
+func newXXHash() hash.Hash {
+	return xxhash.New64()
+}
+
 var hashes = map[string]hashMethod{
 	"md5":    md5.New,
 	"sha1":   sha1.New,
 	"sha256": sha256.New,
 	"sha384": sha512.New384,
 	"sha512": sha512.New,
+	"xxhash": newXXHash,
```

Comment:
How about inlining `xxhash.New64` here instead of creating a new wrapper function, `newXXHash`?

Reply:
Ok I'm done with this

Reply:
Hmm, I'm not seeing your change. Did you forget to push up your commit?

Reply:
Yes, I just pushed all changes. Sorry